src/components/Social/ViewSocial.jsx

```diff
@@ -228,6 +230,103 @@ const MobileSocialIconsContainer = styled.div`
   }
 `
 
+const HeaderText = styled.h1`
+  font-size: 32px;
+  font-weight: 800;
+  margin-top: 1rem;
+
+  ${p => p.theme.mediaQueries.mobile} {
+    font-size: 28px;
+  }
+`
+
+const RecentlyViewedContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+  max-height: 300px;
+  overflow-y: auto;
+  margin-top: -24px;
+
+  ${p => p.theme.mediaQueries.mobile} {
+    max-height: none;
+    overflow-y: visible;
+    justify-content: center;
+    align-items: center;
+    margin-top: -18px;
```

Comment:
```suggestion
    margin-top: -18px;
    width: 100%;
```

to make it a bit less squished 🤏

| Before                                                        | After                                                         |
| :------------------------------------------------------------ | :------------------------------------------------------------ |
| ![2025-03-02 at 10 42PM](https://github.com/user-attachments/assets/adacaf8e-cd55-41d7-b468-1297f6a062b3) | ![2025-03-02 at 10 42PM](https://github.com/user-attachments/assets/0cc4a269-d891-4dc1-8c96-de988f0a2e94) |